code/index.html

```diff
@@ -3,28 +3,116 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Business Site</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+<!-- HEADER-->
+    <header class="header"> 
+      <video playsinline autoplay muted loop >
+        <source src="./videos/video.mp4" type="video/mp4">
+      </video>
+      <div class="overlay">
+        <h1>
+          Book your dream vacation here
+        </h1>
+        <h2>Only one click away</h2>
+        <button>BOOK NOW</button>
```

Comment:
No link here clicking the button, but the page is condensed enough on most all devices that it's already apparent where to go 👍